cerberus/prometheus/client.py

```diff
@@ -16,8 +16,9 @@ def initialize_prom_client(distribution, prometheus_url, prometheus_bearer_token
         prometheus_url = "https://" + url
     if distribution == "openshift" and not prometheus_bearer_token:
         prometheus_bearer_token = runcommand.invoke(
-            "oc -n openshift-monitoring sa get-token prometheus-k8s "
-            "|| oc create token -n openshift-monitoring prometheus-k8s"
+            "oc create token -n openshift-monitoring prometheus-k8s --duration=6h "
```

Comment:
So the token expires after 6 hours? That should be enough in general but should we extend it to 12h just to make sure? Thoughts?

Reply:
I used what we have set up in e2e-benchmarking, but can extend to 12h